src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -14,24 +14,21 @@ export default class UnsupportedBrowserModal extends React.Component<{}, IBrowse
     constructor() {
         super();
 
-        let state;
         if (!browser)  {
```

Comment:
yeah, i think unless we know there's really a condition where this would be true we should remove b/c it implies that's some actual function of the module.  like, if we're running in test, there's no browser or something.  or on a certain device.  when really that thing will never be undefined